java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java

```diff
@@ -15,13 +15,13 @@
 import io.opentracing.propagation.TextMap;
 import io.opentracing.tag.Tags;
 import io.opentracing.util.GlobalTracer;
+import io.vertx.core.Promise;
 import io.vertx.core.buffer.Buffer;
 import io.vertx.ext.web.client.HttpRequest;
+import io.netty.handler.codec.http.*;
+
```

Comment:
extra line

Reply:
Removed

Reply:
We avoid to have `*` when importing classes but listing them all.

Reply:
All wildcard imports removed and replaced with appropriate imports